web/modules/custom/dbcdk_community/src/Form/QuarantineCreateForm.php

```diff
@@ -0,0 +1,177 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\dbcdk_community\Form\QuarantineCreateForm.
+ */
+
+namespace Drupal\dbcdk_community\Form;
+
+use DBCDK\CommunityServices\Api\QuarantineApi;
+use DBCDK\CommunityServices\ApiException;
+use DBCDK\CommunityServices\Model\Profile;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Drupal\Core\DependencyInjection\ContainerInjectionInterface;
+use Drupal\Core\Form\FormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Url;
+
+/**
+ * Create a Community Quarantine.
+ */
+class QuarantineCreateForm extends FormBase implements ContainerInjectionInterface {
+
+  /**
+   * The date format the Community Client expects.
+   */
+  const DATE_FORMAT = 'Y-m-d';
```

Comment:
the DATE_CONSTANT does no longer exist. The forms have been merged to one, but you can find the changes [here](https://github.com/DBCDK/biblo-admin/pull/38/files#diff-f2a84a73f7355c1bbbdd2530354f88e1R122) and in the ::submitForm() of the class.